src/views/css/main.ts

```diff
@@ -64,7 +64,7 @@ const promptGrid = {
     },
 };
 
-const sessionsHeight = `(100vh - ${titleBarHeight + infoPanelHeight}px)`;
+const sessionsHeight = `(100vh - ${titleBarHeight + infoPanelHeight + 5}px)`;
```

Comment:
I have to say that I really wanted to add something more dynamic to it so a font-size change don't effect the result but no combination worked. It seems that 5 is something that needs to be 5